config/ext/rspec.yml

```diff
@@ -356,10 +359,17 @@ RSpec/EmptyLineAfterSubject:
 
 RSpec/EmptyMetadata:
   Description: Avoid empty metadata hash.
-  Enabled: pending
+  Enabled: true
   VersionAdded: '2.24'
+  VersionChanged: '2.31'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/EmptyMetadata
 
+RSpec/EmptyOutput:
```

Comment:
Fand ich gut :+1: 
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecemptyoutput

Reply:
I don't think I have ever needed this, so I'm not sure why I would test for empty output. However, I think I'd prefer disabling it, because when I read  `expect { bar }.to output.to_stderr` I would not expect, that specifically an empty string is meant, but rather that **anything** should have been written to stderr. 
When you compare this for example to the `raise_error` matcher, when you see `expect { do_something }.to raise_error`, it means any error is raised, not one with an empty message. 

TLDR: I'd prefer to more specific, because I think omissions can lead to confusion here.

Reply:
This cop is a detail, but reasonable in my eyes and it should not generate noise. Voting for enabling.

> when I read expect { bar }.to output.to_stderr I would not expect, that specifically an empty string is meant, but rather that anything should have been written to stderr.
When you compare this for example to the raise_error matcher, when you see expect { do_something }.to raise_error, it means any error is raised, not one with an empty message.

I believe this is a misunderstanding. The cop states that "does not print '' to stderr" is equivalent to "prints anything to stderr", which is fine IMO.